tests/baseTestAPI.py

```diff
@@ -176,6 +176,8 @@ def setUpClass(cls):
 
     def setUp(self):
         """Set up broker unit tests."""
+        # Repopulate interfaces if needed
+        self.interfaces = InterfaceHolder()
```

Comment:
InterfaceHolder() is the more general option, if BaseInterface.interfaces is populated it copies that, if not it creates new interface objects.